src/wasm-lib/kcl/src/lint/checks/camel_case.rs

```diff
@@ -0,0 +1,131 @@
+use crate::{
+    ast::types::VariableDeclarator,
+    executor::SourceRange,
+    lint::{
+        rule::{def_finding, Discovered, Finding},
+        Node,
+    },
+};
+
+use anyhow::Result;
+
+def_finding!(
+    Z0001,
```

Comment:
Is the idea is each has a serial number and maybe is related to how users turn lint rules on or off?

Z is for zoo?

Reply:
@Irev-Dev yeah exactly on both points; the code is helpful in case we have multiple ways to run afoul of a rule, and it allows overriding of things. Z is for Zoo (and because the Z00 prefix looks like zoo for the low numbers, which, admittedly is kinda silly :) )

It also helps to print in a standard lint output format to stdout back when I was working on this via the CLI (and the Zoo CLI that I'm going to PR after this lands) to make tooling integration a bit easier